### DISCUSSION TITLE
How to rename dialog toolbar buttons globally?

### DISCUSSION THREAD

**Discussion by rchawla-alphabyte** [serenity-is/Serenity#5955] [Help] [answered]
Hello, I'm wondering how you would change the dialog button titles for "Save and Close" / "Apply Changes Button" GLOBALLY. 

I was able to change the title of the apply-changes-button, however not I'm not able to change the title of the save-and-close-button?

Any help would be appreciated!

![changebuttonsglobally](https://user-images.githubusercontent.com/76563021/131678382-8297c7b7-c786-4df7-bdb1-19f7fa5438a6.png)

**Comment by VictorTomaili** [comment, marked as answer]
Just use local texts

![image](https://user-images.githubusercontent.com/2023440/131679082-87274430-b4f2-4472-acc7-027c19f2d382.png)

**Reply by rchawla-alphabyte** [reply]
This works for the save-and-close-button, however I cannot see the updated text for the apply changes button

![image](https://user-images.githubusercontent.com/76563021/131680366-0c362300-385c-42ef-af1d-a1d41f9b30a7.png)
![image](https://user-images.githubusercontent.com/76563021/131680450-d821ec20-1a1f-42cd-82d7-1a8b9a0e716a.png)

**Comment by rchawla-alphabyte** [comment]
Never mind thanks Victor! I fixed it by adding the title in the Serenity.CoreLib.js (it only had the hint attribute)

![image](https://user-images.githubusercontent.com/76563021/131681342-de5ec255-0edc-4fab-b93b-890a3370fba2.png)
